microhapulator/mix.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+#
+# -----------------------------------------------------------------------------
+# Copyright (c) 2019, Battelle National Biodefense Institute.
+#
+# This file is part of MicroHapulator (github.com/bioforensics/microhapulator)
+# and is licensed under the BSD license: see LICENSE.txt.
+# -----------------------------------------------------------------------------
+
+import microhapulator
+from microhapulator.genotype import SimulatedGenotype
+
+
+def main(args):
```

Comment:
The sole purpose of this module is to expose the functionality to the command-line interface. At the Python API level, it can be handled by a single simple command.